src/App.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import { Board } from './components/Board/Board';
 import './App.scss';
-import store from './app/store';
+import store from './store/store';
 import { DndProvider } from 'react-dnd';
 import { HTML5Backend } from 'react-dnd-html5-backend';
+import { Board } from 'components/Board/Board';
```

Comment:
It would be nice if you add to every component index.ts file that reexports component or you can name in component folder as index.tsx, but on projects we prefer first variation. Its all for import like 'components/Board'